fineract-provider/src/main/java/org/apache/fineract/adhocquery/domain/AdHoc.java

```diff
@@ -55,23 +62,8 @@ public final class AdHoc extends AbstractAuditableCustom {
     @Column(name = "report_run_every")
     private Long reportRunEvery;
 
-    @Column(name = "is_active", nullable = false)
-    private boolean isActive = false;
-
-    private AdHoc() {}
-
-    private AdHoc(final String name, final String query, final String tableName, final String tableFields, final String email,
-            final Long reportRunFrequency, final Long reportRunEvery, final boolean isActive) {
-        this.name = StringUtils.defaultIfEmpty(name, null);
-        this.query = StringUtils.defaultIfEmpty(query, null);
-        this.tableName = StringUtils.defaultIfEmpty(tableName, null);
-        this.tableFields = StringUtils.defaultIfEmpty(tableFields, null);
-        this.email = StringUtils.defaultIfEmpty(email, null);
-        this.reportRunFrequency = reportRunFrequency;
-        this.reportRunEvery = reportRunEvery;
-        this.isActive = BooleanUtils.toBooleanDefaultIfNull(isActive, false);
-
-    }
+    @Column(name = "active", nullable = false)
```

Comment:
This is not gonna fly, the database migration is missing for the column renaming.